.gitlab/binary_build/cluster_agent.yml

```
@@ -4,7 +4,7 @@
   needs: ["go_mod_tidy_check"]
   script:
     # Assuming that by default, the RELEASE_VERSION_7 entry will get used (ie. nightly-a7)
-    - if [[ "$CI_COMMIT_TAG" =~ ^dca- ]]; then export RELEASE_VERSION="$CI_COMMIT_TAG"; else export RELEASE_VERSION="$RELEASE_VERSION_7"; fi
+    - if [[ "$CI_COMMIT_TAG" =~ /^[0-9]+\.[0-9]+\.[0-9]+-rc\.[0-9]+$/ ]]; then export RELEASE_VERSION="$CI_COMMIT_TAG"; else export RELEASE_VERSION="$RELEASE_VERSION_7"; fi
```

This logic applied before because we had `dca-x.y.z` entries in `release.json`. I assume this is not going to be the case in the future anymore, so we can always use the Agent 7 release entry:
```suggestion
    - export RELEASE_VERSION="$RELEASE_VERSION_7"
```